repo/packages/M/marathon/3/config.json

```diff
@@ -0,0 +1,287 @@
+        },
+        "mem": {
+          "default": 1024.0,
+          "description": "Memory (MB) to allocate to each Marathon task.",
+          "minimum": 512.0,
+          "type": "number"
+        },
+        "mesos-authentication-principal": {
+          "description": "The Mesos principal used for authentication.",
+          "type": "string"
+        },
+        "mesos-authentication-secret-file": {
+          "description": "The path to the Mesos secret file containing the authentication secret.",
+          "type": "string"
+        },
+        "mesos-role": {
+          "description": "Mesos role for this framework.",
+          "type": "string"
+        },
+        "min-revive-offers-interval": {
+          "default": 5000,
+          "description": "Do not ask for all offers (also already seen ones) more often than this interval (ms).",
+          "minimum": 200,
+          "type": "integer"
+        },
+        "reconciliation-initial-delay": {
+          "description": "The delay, in milliseconds, before Marathon begins to periodically perform task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "reconciliation-interval": {
+          "description": "The period, in milliseconds, between task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "revive-offers-for-new-apps": {
+          "default": true,
+          "description": "Whether to call reviveOffers for new or changed apps.",
+          "type": "boolean"
+        },
+        "scale-apps-initial-delay": {
+          "default": 15000,
+          "description": "This is the length of time, in milliseconds, before Marathon begins to periodically attempt to scale apps",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "scale-apps-interval": {
+          "default": 300000,
+          "description": "This is the length of time, in milliseconds, between task scale apps.",
+          "minimum": 1000,
+          "type": "integer"
+        },
+        "ssl-keystore-password": {
+          "description": "Password for the keystore supplied with the marathon/ssl-keystore-path option.",
+          "type": "string"
+        },
+        "ssl-keystore-path": {
+          "description": "Path to the SSL keystore. SSL will be enabled if this option is supplied. The keystore can be downloaded into Marathon's working directory by using the uris property.",
+          "type": "string"
+        },
+        "uris": {
+          "default": [],
+          "description": "List of URIs that will be download and made available in the current working directory of Marathon. For example this can be used to download a Java keystore file for SSL configuration.",
+          "items": {
+            "pattern": "^[\\s]+",
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "zk": {
+          "default": "zk://master.mesos:2181/universe",
```

Comment:
I completely agree! I just copied it from the other config file.

It would be even better to base the ZK path on the app-id or something like that (if it is possible).

Unfortunately, I think its even worse to change the default now. If someone does upgrade from a prior package definition, the Marathon will not contain the old definitions anymore.

Do you agree or can you think of a way to mitigate that?